src/main/java/com/autilus/hw2806/MailService.java

```diff
@@ -0,0 +1,27 @@
+package com.autilus.hw2806;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public static class MailService<T> implements Consumer<Send<T>> {
+    @Override
+    public void accept(Send<T> t) {
+        List<T> v = box.get(t.getTo());
+        v.add(t.getContent());
+        box.put(t.getTo(), v);
+    }
+
+    public MailService() {
+        box = (HashMap) get(key) ->{ return super.getOrDefault(key, new LinkedList<T>()); };
```

Comment:
we can change: 
new LinkedList<T>() -> Collections.emptyList()

Reply:
in intellij it works with Collections.emptyList(), but in stepic not(

<img width="582" alt="Screen Shot 2019-07-02 at 6 19 50 PM" src="https://user-images.githubusercontent.com/25722810/60524858-fe5dee00-9cf5-11e9-8258-501bf8917bbe.png">